### ISSUE TITLE
Separate Audio Implementation

### ISSUE THREAD

**Issue by IsidorosT** [laggingreflex/multi-video-player#3]
Hi @laggingreflex,
I saw your multi-video player and it's a great job.

I have a question. What about if we wanted to add separate audio playing for those videos?

**Comment by laggingreflex** [comment]
Thanks & glad you found it useful!

Not sure I understand, could you elaborate please? What do you mean by separate audios?

**Comment by IsidorosT** [comment]
What If someone want to drop a sound file(.wav) to play at the background also

**Comment by laggingreflex** [comment]
I just tested it, it seems to be working for me. But it's hard to see it visibly on screen since it has no video... if you hover over it'll show an empty player with controls.

**Comment by IsidorosT** [comment]
Yeah i tested it and you are right. The thing we try is to have an audio and throw videos at current timeframe of the audio,
